### ISSUE TITLE
node-polyglot is not in package.json

### ISSUE THREAD

**Issue by carloscarcamo** [JaredHawkins/TweetGeoViz#59]
I have this error when doing ```npm run client-start```
```
ERROR in ./client/src/stores/languageStore.js
Module not found: Error: Cannot resolve module 'node-polyglot' in /path/to/TweetGeoViz/client/src/stores
 @ ./client/src/stores/languageStore.js 49:15-39
```
It works after doing ```npm install node-polyglot```

This module should be in dependencies or in devDependencies?

**Comment by anvk** [comment]
My bad.... I might've forgotten to add it to the package.json file.
It is definitely a regular dependency since it is a library to handle translation tree.

**Comment by anvk** [comment]
Should be fixed in 0.2.5
